kyuubi-server/src/main/scala/org/apache/kyuubi/server/KyuubiRestFrontendService.scala

```diff
@@ -174,14 +174,14 @@ class KyuubiRestFrontendService(override val serverable: Serverable)
         server.start()
         recoverBatchSessions()
         isStarted.set(true)
-        info(s"$getName has started at ${server.getServerUri}")
         startBatchChecker()
         startInternal()
       } catch {
         case e: Exception => throw new KyuubiException(s"Cannot start $getName", e)
       }
     }
     super.start()
+    info(s"Exposing REST endpoint at: ${server.getServerUri}")
```

Comment:
Make output consistent
```
KyuubiTBinaryFrontendService: Service[KyuubiTBinaryFrontend] is started.
KyuubiTBinaryFrontendService: Starting and exposing JDBC connection at: jdbc:hive2://10.221.96.184:10009/
...
KyuubiRestFrontendService: Service[KyuubiRestFrontendService] is started.
KyuubiRestFrontendService: Exposing REST endpoint at: 10.221.96.184:10099
```

Reply:
Possible to output the protocol as well?

Reply:
OK, currently it does not support SSL, so simply added `http://`